core/src/main/java/com/opensymphony/xwork2/config/impl/DefaultConfiguration.java

```diff
@@ -385,6 +386,7 @@ protected Container createBootstrapContainer(List<ContainerProvider> providers)
         builder.factory(ExpressionCacheFactory.class, DefaultOgnlExpressionCacheFactory.class, Scope.SINGLETON);
         builder.factory(BeanInfoCacheFactory.class, DefaultOgnlBeanInfoCacheFactory.class, Scope.SINGLETON);
         builder.factory(OgnlUtil.class, Scope.SINGLETON);
+        builder.factory(SecurityMemberAccess.class, Scope.PROTOTYPE);
```

Comment:
Given it's `prototype` scope, I don't believe it to make any difference

Reply:
That is, the container would return a new instance, and not the one that is being used by the ValueStack

Reply:
I meant it will be possible creating a new instance via `container`s methods, yet we excluded the whole package `com.opensymphony.xwork2.inject` which means attacker cannot use container in the first place.

Reply:
True, I also don't think there's anything useful an attacker can do with a fresh `SecurityMemberAccess` instance